### PR TITLE
[terra-tabs] Fixes tabs padding, box-shadow and tab large content display

### PR DESCRIPTION
**Pull request by sugan2416**
### Summary

**What was changed:**

- Additional padding of 10px on the left and right of tab content body is removed.
- Box shadow container style removed for framework variant tabs
- Fixes large tab content display to fix tab header in its position with a scrollable view.  

**Why it was changed:**

- Padding styles needed to switched between framework and workspace variants.
- Box shadow style needed to switched between framework and workspace variants. 
- Large content overflow needed to handled to allow visibility of tab header during scroll.

### Testing

This change was tested using:

- [ ] WDIO
- [ ] Jest
- [ ] Visual testing (please attach a screenshot or recording)
- [ ] Other (please describe below)
- [x] No tests are needed

### Reviews

In addition to engineering reviews, this PR needs:

- [ ] UX review
- [ ] Accessibility review
- [ ] Functional review

### Additional Details

**This PR resolves:**

UXPLATFORM-9671 

---

Thank you for contributing to Terra.
@cerner/terra

Before
image

After
image

Before
image

After
image

### CODE REVIEW

File: packages/terra-tabs/src/common-tabs/CommonTabContent.module.scss [hunk 1]
```diff
@@ -8,16 +8,22 @@
...
     width: 100%;
   }
 
+  .panel-padding {
+    padding: 0 10px;
+  }
+
   .panel-header {
     flex: 0 0 auto;
+    z-index: 10;
```
Comment (supreethmr) [inline packages/terra-tabs/src/common-tabs/CommonTabContent.module.scss:21]: curious, how did we arrive at z-index 10..??

**Comment by supreethmr** [comment]
Are these changes tested on the applications where the issues were identified initially.

File: packages/terra-tabs/src/common-tabs/CommonTabContent.module.scss (hunk 1 above)
Comment (sugan2416) [reply packages/terra-tabs/src/common-tabs/CommonTabContent.module.scss:21]: this is an existing content style (from workspace) seen in https://github.com/cerner/terra-framework/pull/1807/files#diff-133172bdc7e9047a876b59a63ac335d93cdefd591f1412bfd70c64d716fe722aL20, the change made was to move padding under panel-heading-padding retaining z-index as it is under panel-header.

**Comment by sugan2416** [comment]
> Are these changes tested on the applications where the issues were identified initially.

Yes, these three changes were tested in him-providers-view-js (reported issue application). Padding and large content fix can be observed in dev-site examples.

**Review by supreethmr — APPROVED** [review]

**Review by saket2403 — APPROVED** [review]

**Pull request merged**